apps/Gruntfile.js

```diff
@@ -862,7 +867,12 @@ describe('entry tests', () => {
     var current = path.resolve('build/locale/current');
     child_process.execSync('mkdir -p ' + current);
     appsToBuild
-      .concat('common', 'tutorialExplorer', 'regionalPartnerSearch')
+      .concat(
+        'common',
+        'tutorialExplorer',
+        'regionalPartnerSearch',
+        'regionalPartnerMiniContact'
+      )
```

Comment:
@davidsbailey You've got the most recent knowledge on JS build configuration - does this make sense?

Reply:
no -- this doesn't look right, because only locale-related files should be added here. were you getting an error without adding these here, @breville ?

Reply:
sorry... I misread. seems like you are just adding `'regionalPartnerMiniContact'` which follows the existing pattern, so this looks good to me. I'm not sure what a better pattern would be but at least this way we only have one pattern to clean up later (if that turns out to be needed).

so this line LGTM